src/common/containers/landing.js

```diff
@@ -27,7 +27,7 @@ class Landing extends Component {
 
               <ul className={ styles.banner }>
                 <li className={ styles.bannerImage }>
-                  <img src='https://assets.ubuntu.com/v1/ed6d1c5b-build.snapcraft.hero.svg' alt="Snapcraft hero image, push to GitHub, built automatically, release for your users."/>
+                  <img src='https://assets.ubuntu.com/v1/ed6d1c5b-build.snapcraft.hero.svg' alt=""/>
```

Comment:
@richmccartney Unfortunately, that article is completely incorrect. It repeatedly refers to `alt` text as a “description” or “describing” an image, [directly contradicting the HTML specification](https://html.spec.whatwg.org/multipage/images.html#a-phrase-or-paragraph-with-an-alternative-graphical-representation:-charts,-diagrams,-graphs,-maps,-illustrations): “It is important to realize that the alternative text is a *replacement* for the image, not a description of the image”.

In this case, as I said in the PR, the `alt` text I removed was “redundant with text immediately below”. This is exactly [as recommended by the HTML spec](https://html.spec.whatwg.org/multipage/images.html#general-guidelines): “the `alt` attribute's value should not repeat information that is already provided in the prose next to the image.”